shared/src/common/types/interview/interviews-get-items-response-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+import { InterviewResponseDto } from './interview-get-response-dto.type';
+
+type InterviewsResponseDto = {
```

Comment:
interviews-get-all-response-dto.type.ts
```suggestion
type InterviewsGetAllResponseDto = {
```